shell/browser/api/electron_api_browser_view.h

```diff
@@ -7,11 +7,14 @@
 
 #include <memory>
 #include <string>
+#include <vector>
 
 #include "content/public/browser/web_contents_observer.h"
 #include "gin/handle.h"
 #include "gin/wrappable.h"
+#include "shell/browser/api/electron_api_web_contents.h"
```

Comment:
`shell/browser/api/electron_api_web_contents.h` not needed

instead `shell/browser/extended_web_contents_observer.h`

Reply:
is this not needed for https://github.com/electron/electron/blob/9bd0fc534809e5ab61c73fbcadc6713e4e23dd44/shell/browser/api/electron_api_browser_view.h#L50?

Reply:
it is forward declared below https://github.com/electron/electron/pull/26145/files#diff-26667b9567af60d3a0b31066cb0c35e878e90f16a285c67d46390812ac0940dbR36

Reply:
ah missed that! thanks, done 🙇🏻‍♀️